src/basic_operations.jl

```diff
@@ -18,6 +18,22 @@
 @inline Base.:*(S::AbstractTensor, n::Number) = _map(x -> x*n, S)
 @inline Base.:*(n::Number, S::AbstractTensor) = _map(x -> n*x, S)
 @inline Base.:/(S::AbstractTensor, n::Number) = _map(x -> x/n, S)
+function Base.:^(S::SecondOrderTensor, p::Int)
+    if p == 1
+        return S
+    elseif p == 0
+        return one(S)
+    elseif p == -1
+        return inv(S)
+    elseif p < 0
```

Comment:
I would add more error message just in case.

Stacktrace:
 [1] macro expansion at /home/travis/.julia/v0.6/Tensors/test/test_misc.jl:42 [inlined]
 [2] macro expansion at ./test.jl:853 [inlined]
 [3] macro expansion at /home/travis/.julia/v0.6/Tensors/test/runtests.jl:9 [inlined]
 [4] macro expansion at /home/travis/.julia/v0.6/TimerOutputs/src/TimerOutput.jl:128 [inlined]
 [5] macro expansion at /home/travis/.julia/v0.6/Tensors/test/runtests.jl:8 [inlined]
 [6] anonymous at ./<missing>:?